bom/runtime/pom.xml

```diff
@@ -114,7 +114,7 @@
         <derby-jdbc.version>10.14.2.0</derby-jdbc.version>
         <shrinkwrap.version>1.2.6</shrinkwrap.version>
         <rest-assured.version>4.1.2</rest-assured.version>
-        <junit.jupiter.version>5.5.2</junit.jupiter.version>
+        <junit.jupiter.version>5.6.0-RC1</junit.jupiter.version>
```

Comment:
https://github.com/quarkusio/quarkus/pull/6663 updates to 5.6.0. It would be nice to rebase once that PR is merged